examples/patterns/forms/form-stacked.html

```diff
@@ -11,6 +11,27 @@
       <input type="text" id="full-name-stacked" required>
     </div>
   </div>
+  <div class="p-form__group">
+    <label for="username-stacked" class="p-form__label">Username</label>
+    <div class="p-form__control">
+        <input type="text" id="username-stacked">
+        <p class="p-form-help-text">
+            Lowercase alphanumeric characters and - only.
+        </p>
+    </div>
+  </div>
+  <div class="p-form__group p-form-validation is-error">
+      <label for="username-stacked-error" class="p-form__label">Username</label>
+      <div class="p-form__control">
+          <input type="text" id="username-stacked-error" class="p-form-validation__input" aria-invalid="true" aria-describedby="username-error-message-stacked">
+          <p class="p-form-help-text">
+              Lowercase alphanumeric characters and - only.
+          </p>
+          <p class="p-form-validation__message" id="username-error-message-stacked" role="alert">
```

Comment:
I thought validation should come before the help text?

Reply:
You're right, done